lib/plugin-manager.js

```diff
@@ -0,0 +1,303 @@
+/* global Log, Config */
+'use strict';
+
+const EventEmitter = require('events').EventEmitter;
+const async = require('async');
+const StringTemplate = require('./string-template');
+const Metadata = require('./source/metadata');
+const S3 = require('./source/s3');
+
+// const Consul = require('./source/consul');
+// const File = require('./source/file');
```

Comment:
Probably worth a "TODO" comment that these are in progress. Or just take out the commented lines.